src/schema/__tests__/sale_artworks.test.js

```diff
@@ -10,6 +11,48 @@ describe("Sale Artworks", () => {
     })
   }
 
+  it("pulls from /sale_artworks endpoint if `live_sale=true` and `include_lots_by_followed_artists=true`", async () => {
+    const hits = _.fill(Array(10), { id: "foo" })
+    const totalCount = hits.length * 2
+    const gravityResponse = hits
+    const query = gql`
+      {
+        sale_artworks(
+          live_sale: true
+          include_artworks_by_followed_artists: true
+        ) {
+          counts {
+            total
+          }
+          edges {
+            node {
+              id
+            }
+          }
+        }
+      }
+    `
+
+    schema.__Rewire__("getTotal", () =>
+      Promise.resolve({
+        body: {
+          total: totalCount,
+        },
+      })
+    )
+
+    const { sale_artworks: { counts: { total }, edges } } = await execute(
+      gravityResponse,
+      query,
+      {
+        saleArtworksAllLoader: () => Promise.resolve(gravityResponse),
+      }
+    )
+    expect(total).toEqual(totalCount)
+    expect(edges.length).toEqual(hits.length)
+    schema.__ResetDependency__("getTotal")
+  })
+
```

Comment:
Since we're utilizing two endpoints depending on whether or not `live_sale: true` etc is passed in technically all of these tests need to be duplicated.